styles/properties.less

```diff
@@ -60,6 +60,8 @@
   }
 
   [contenteditable] {
+    min-height: 24px;
```

Comment:
Why is this change needed?

Reply:
Just a quick fix for this issue:

![18-09-_2020_11-06-04](https://user-images.githubusercontent.com/7633572/93579657-08254d00-f99f-11ea-9546-3a65aad5b022.gif)

Reply:
Alright, nice finding 👍